github-action/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo "::debug::Using input path: ${INPUT_INPUT_PATH}"
```

Comment:
Not sure why the term `INPUT` precedes all of these variable names, especially when you have something like `INPUT_OUTPUT_PATH` that seems to just lend to confusion.  I would suggest we just remove the extra leading `INPUT` for all of these var names.

Reply:
Yeah, it's somewhat unpleasant, but GH actions prefixes all configured inputs with `INPUT_` and then exposes them as ENV variables. To the end user of the action though, they would just configure `input_path`, which maps well to the CLI naming. I could change it to `path` but then it would result in that disconnect.